tests/test_formatters.py

```diff
@@ -0,0 +1,111 @@
+import pandas as pd
+import pytest
+
+from buster.formatters.documents import DocumentsFormatter
+from buster.formatters.prompts import SystemPromptFormatter
+from buster.tokenizers import GPTTokenizer
+
+
+def test_documents_formatter__normal():
+    tokenizer = GPTTokenizer(model_name="gpt-3.5-turbo")
+    documents_formatter = DocumentsFormatter(
+        tokenizer=tokenizer,
+        max_tokens=100,
+    )
+
+    # In this test, document_1 doesn't entirely fit.
+    # we only expect a part of it to be contained.
+
+    document_1 = "This is a very short document."
+    document_2 = "This is another very short document."
+    document_3 = "This is also a short document."
+
+    matched_documents = pd.DataFrame({"content": [document_1, document_2, document_3]})
+
+    docs_str, matched_documents_new = documents_formatter.format(matched_documents)
+
+    # less documents and the new document is shorter than the original
+    matched_documents.content == matched_documents_new.content
```

Comment:
there is no assert, is this actually doing something ?

Reply:
youre right, it wasnt, added an `assert all()`